### PR TITLE
rio: Add version 0.0.36

### PR DESCRIPTION
**Pull request by Pkcarreno**
Closes https://github.com/ScoopInstaller/Extras/issues/11772

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aliesbelik** [comment]
Wonder if you could use already tested and tuned manifest from here:
https://github.com/melMass/scoop-mtb/blob/master/bucket/rio.json
to add add options you want.

**Comment by Pkcarreno** [comment]
Hey @aliesbelik, 
after uploading the first version of the manifest I saw that repo and it helped me to solve the error of not showing the icon.

Credit to that guy for that, overall it's a pretty simple manifest.

File: bucket/rio.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.0.36",
+    "description": "A hardware-accelerated GPU terminal emulator focusing to run in desktops and browsers",
+    "homepage": "https://github.com/raphamorim/rio",
+    "license": "MIT",
+    "url": [
+        "https://github.com/raphamorim/rio/releases/download/v0.0.36/Rio-portable.exe#/rio.exe",
+        "https://raw.githubusercontent.com/raphamorim/rio/main/misc/windows/rio.ico"
+    ],
+    "hash": "98d09f4774a001027519bc5f1c711543ca62dc4411343a60e0551b3d51cc5264",
```
Comment (aliesbelik) [inline bucket/rio.json:10]:
```suggestion
    "hash": [
        "98d09f4774a001027519bc5f1c711543ca62dc4411343a60e0551b3d51cc5264",
        "dab2e4897add153ee287a92a8380a35a8fd0e4932f63fec7405b70917e6d6d4a"
    ],
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

File: bucket/rio.json (hunk 1 above)
Comment (Pkcarreno) [reply bucket/rio.json:10]: Nice Catch!

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**